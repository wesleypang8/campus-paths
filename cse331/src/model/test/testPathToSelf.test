#test path to self

r
CSE
CSE

# All of your test scripts should end with q to terminate the program.
q