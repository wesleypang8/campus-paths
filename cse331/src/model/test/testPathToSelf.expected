Menu:
    r to find a route
    b to see a list of all buildings
    q to quit

Enter an option ('m' to see the menu): #test path to self

Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to Paul G. Allen Center for Computer Science & Engineering:
Total distance: 0 feet

Enter an option ('m' to see the menu): 
# All of your test scripts should end with q to terminate the program.
